qemu.mk

```diff
@@ -79,10 +79,12 @@ busybox-cleaner: busybox-cleaner-common
 ################################################################################
 # Linux kernel
 ################################################################################
-$(LINUX_PATH)/.config:
+$(LINUX_PATH)/.config: kconfig_preempt.conf
```

Comment:
Does the target `kconfig_preempt.conf` really make any sense here? I mean this makefile isn't in `$(LINUX_PATH)` when being called. I.e, shouldn't it be `$(LINUX_PATH)/kconfig_preempt.conf`?

However, I have test built using it as it is and indeed it is working.